### ISSUE TITLE
hash mismatch for orjson source

### ISSUE THREAD

**Issue by das-g** [nix-community/poetry2nix#934]
### Describe the issue

To reproduce:
```bash
cd $(mktemp -d)
nix run nixpkgs#poetry -- init --name=foo --dev-dependency=orjson -n
nix run nixpkgs#poetry -- lock

cat << EOF > shell.nix
{ pkgs ? import <nixpkgs> { }
, poetry2nix ? import ~/dev/poetry2nix { }
}:
let
  myAppEnv = poetry2nix.mkPoetryEnv {
    projectDir = ./.;
    editablePackageSources = {
      my-app = ./src;
    };
  };
in myAppEnv.env
EOF

nix-shell --command 'python -c "import orjson"'
```
#### expected
Command succeeds (just like `(cd $(mktemp -d) && nix-shell -p 'python3.withPackages (ps: [ps.orjson])' --command 'python -c "import orjson"')` does)

#### observed
<details><summary><code>error: hash mismatch in fixed-output derivation '/nix/store/l2ivjcs7xmifvcimi7p06wymc8vf29kj-orjson-3.8.4-vendor.tar.gz.drv'</code> (click to expand)</summary>

Terminal output:
```
trace: warning: Unknown orjson version: '3.8.4'. Please update getCargoHash.
these 4 derivations will be built:
  /nix/store/l2ivjcs7xmifvcimi7p06wymc8vf29kj-orjson-3.8.4-vendor.tar.gz.drv
  /nix/store/pcv2ddfb3hk2h5d991vd4knwl45nbl01-python3.10-orjson-3.8.4.drv
  /nix/store/wk27ww2lkxrqzxkxbjswk95i4hdavcbn-foo-editable.drv
  /nix/store/9ka7ch657qimjvg2npf5g5mr3br4vwyl-python3-3.10.9-env.drv
these 12 paths will be fetched (4.93 MiB download, 27.21 MiB unpacked):
  /nix/store/3v42h3hw10mfxncivipfn84zbfpgff08-python-remove-tests-dir-hook
  /nix/store/5yvrs7rs25w8jzwv3wg93h20q59z4yy9-fixup-hook.sh
  /nix/store/696x69nrqw4d8mclxs2gf53apikp3ig6-python3.10-pip-22.2.2
  /nix/store/6zh035j7znhpb33qcxyg8jbxxham5hry-python3.10-setuptools-65.3.0
  /nix/store/8xkwld47p3yai11d0pk0vmg65qnkbwkb-python-namespaces-hook.sh
  /nix/store/b4j2gs2mpx577wp75b5c5gp2iwb9m9y3-pip-install-hook
  /nix/store/bvm702bws1fcx1scp242y28xzrn8cdci-hook
  /nix/store/c82584k8nk9wr5h2as47kd6m9hciwisd-pip-build-hook.sh
  /nix/store/g7rrxc7yy92dpvp2fadkdr9qzzky3ki1-python3.10-pip-22.2.2
  /nix/store/m9jfnxmkw0132y48jc9lpngkx0djdr7l-python-catch-conflicts-hook
  /nix/store/vzl2k8xs9ykcc8adhsyd2gh872qv8j79-python3.10-wheel-0.37.1
  /nix/store/xqr3kzf63hijk8g4apafwig2j0slg95x-python-imports-check-hook.sh
copying path '/nix/store/5yvrs7rs25w8jzwv3wg93h20q59z4yy9-fixup-hook.sh' from 'https://cache.nixos.org'...
copying path '/nix/store/3v42h3hw10mfxncivipfn84zbfpgff08-python-remove-tests-dir-hook' from 'https://cache.nixos.org'...
building '/nix/store/wk27ww2lkxrqzxkxbjswk95i4hdavcbn-foo-editable.drv'...
copying path '/nix/store/bvm702bws1fcx1scp242y28xzrn8cdci-hook' from 'https://cache.nixos.org'...
copying path '/nix/store/xqr3kzf63hijk8g4apafwig2j0slg95x-python-imports-check-hook.sh' from 'https://cache.nixos.org'...
copying path '/nix/store/8xkwld47p3yai11d0pk0vmg65qnkbwkb-python-namespaces-hook.sh' from 'https://cache.nixos.org'...
copying path '/nix/store/6zh035j7znhpb33qcxyg8jbxxham5hry-python3.10-setuptools-65.3.0' from 'https://cache.nixos.org'...
copying path '/nix/store/696x69nrqw4d8mclxs2gf53apikp3ig6-python3.10-pip-22.2.2' from 'https://cache.nixos.org'...
copying path '/nix/store/g7rrxc7yy92dpvp2fadkdr9qzzky3ki1-python3.10-pip-22.2.2' from 'https://cache.nixos.org'...
copying path '/nix/store/vzl2k8xs9ykcc8adhsyd2gh872qv8j79-python3.10-wheel-0.37.1' from 'https://cache.nixos.org'...
building '/nix/store/l2ivjcs7xmifvcimi7p06wymc8vf29kj-orjson-3.8.4-vendor.tar.gz.drv'...
unpacking sources
unpacking source archive /nix/store/jipxylw90zzis5wiy4fclycclbc3y5wd-orjson-3.8.4.tar.gz
source root is orjson-3.8.4
setting SOURCE_DATE_EPOCH to timestamp 1672847233 of file orjson-3.8.4/test/util.py
patching sources
configuring
no configure script, doing nothing
building
copying path '/nix/store/m9jfnxmkw0132y48jc9lpngkx0djdr7l-python-catch-conflicts-hook' from 'https://cache.nixos.org'...
    Updating crates.io index
copying path '/nix/store/c82584k8nk9wr5h2as47kd6m9hciwisd-pip-build-hook.sh' from 'https://cache.nixos.org'...
copying path '/nix/store/b4j2gs2mpx577wp75b5c5gp2iwb9m9y3-pip-install-hook' from 'https://cache.nixos.org'...
 Downloading crates ...
  Downloaded beef v0.5.2
  Downloaded bytecount v0.6.3
  Downloaded associative-cache v1.0.1
  Downloaded arrayvec v0.7.2
  Downloaded itoap v1.0.1
  Downloaded libm v0.1.4
  Downloaded packed_simd_2 v0.3.8
  Downloaded smallvec v1.10.0
  Downloaded simdutf8 v0.1.4
  Downloaded castaway v0.2.2
  Downloaded ahash v0.8.2
  Downloaded num-integer v0.1.45
  Downloaded target-lexicon v0.12.5
  Downloaded ryu v1.0.12
  Downloaded serde v1.0.152
  Downloaded serde_json v1.0.91
  Downloaded version_check v0.9.4
  Downloaded rustversion v1.0.11
  Downloaded chrono v0.4.23
  Downloaded cc v1.0.78
  Downloaded pyo3-ffi v0.17.3
  Downloaded once_cell v1.17.0
  Downloaded libc v0.2.139
  Downloaded encoding_rs v0.8.31
  Downloaded itoa v1.0.5
  Downloaded cfg-if v1.0.0
  Downloaded autocfg v1.1.0
  Downloaded pyo3-build-config v0.17.3
  Downloaded num-traits v0.2.15
  Downloaded compact_str v0.6.1
   Vendoring ahash v0.8.2 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/ahash-0.8.2) to orjson-3.8.4-vendor.tar.gz/ahash
   Vendoring arrayvec v0.7.2 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/arrayvec-0.7.2) to orjson-3.8.4-vendor.tar.gz/arrayvec
   Vendoring associative-cache v1.0.1 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/associative-cache-1.0.1) to orjson-3.8.4-vendor.tar.gz/associative-cache
   Vendoring autocfg v1.1.0 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/autocfg-1.1.0) to orjson-3.8.4-vendor.tar.gz/autocfg
   Vendoring beef v0.5.2 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/beef-0.5.2) to orjson-3.8.4-vendor.tar.gz/beef
   Vendoring bytecount v0.6.3 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/bytecount-0.6.3) to orjson-3.8.4-vendor.tar.gz/bytecount
   Vendoring castaway v0.2.2 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/castaway-0.2.2) to orjson-3.8.4-vendor.tar.gz/castaway
   Vendoring cc v1.0.78 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/cc-1.0.78) to orjson-3.8.4-vendor.tar.gz/cc
   Vendoring cfg-if v1.0.0 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/cfg-if-1.0.0) to orjson-3.8.4-vendor.tar.gz/cfg-if
   Vendoring chrono v0.4.23 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/chrono-0.4.23) to orjson-3.8.4-vendor.tar.gz/chrono
   Vendoring compact_str v0.6.1 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/compact_str-0.6.1) to orjson-3.8.4-vendor.tar.gz/compact_str
   Vendoring encoding_rs v0.8.31 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/encoding_rs-0.8.31) to orjson-3.8.4-vendor.tar.gz/encoding_rs
   Vendoring itoa v1.0.5 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/itoa-1.0.5) to orjson-3.8.4-vendor.tar.gz/itoa
   Vendoring itoap v1.0.1 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/itoap-1.0.1) to orjson-3.8.4-vendor.tar.gz/itoap
   Vendoring libc v0.2.139 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/libc-0.2.139) to orjson-3.8.4-vendor.tar.gz/libc
   Vendoring libm v0.1.4 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/libm-0.1.4) to orjson-3.8.4-vendor.tar.gz/libm
   Vendoring num-integer v0.1.45 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/num-integer-0.1.45) to orjson-3.8.4-vendor.tar.gz/num-integer
   Vendoring num-traits v0.2.15 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/num-traits-0.2.15) to orjson-3.8.4-vendor.tar.gz/num-traits
   Vendoring once_cell v1.17.0 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/once_cell-1.17.0) to orjson-3.8.4-vendor.tar.gz/once_cell
   Vendoring packed_simd_2 v0.3.8 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/packed_simd_2-0.3.8) to orjson-3.8.4-vendor.tar.gz/packed_simd_2
   Vendoring pyo3-build-config v0.17.3 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/pyo3-build-config-0.17.3) to orjson-3.8.4-vendor.tar.gz/pyo3-build-config
   Vendoring pyo3-ffi v0.17.3 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/pyo3-ffi-0.17.3) to orjson-3.8.4-vendor.tar.gz/pyo3-ffi
   Vendoring rustversion v1.0.11 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/rustversion-1.0.11) to orjson-3.8.4-vendor.tar.gz/rustversion
   Vendoring ryu v1.0.12 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/ryu-1.0.12) to orjson-3.8.4-vendor.tar.gz/ryu
   Vendoring serde v1.0.152 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/serde-1.0.152) to orjson-3.8.4-vendor.tar.gz/serde
   Vendoring serde_json v1.0.91 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/serde_json-1.0.91) to orjson-3.8.4-vendor.tar.gz/serde_json
   Vendoring simdutf8 v0.1.4 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/simdutf8-0.1.4) to orjson-3.8.4-vendor.tar.gz/simdutf8
   Vendoring smallvec v1.10.0 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/smallvec-1.10.0) to orjson-3.8.4-vendor.tar.gz/smallvec
   Vendoring target-lexicon v0.12.5 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/target-lexicon-0.12.5) to orjson-3.8.4-vendor.tar.gz/target-lexicon
   Vendoring version_check v0.9.4 (/build/orjson-3.8.4/cargo-home.iQE/registry/src/github.com-1ecc6299db9ec823/version_check-0.9.4) to orjson-3.8.4-vendor.tar.gz/version_check
To use vendored sources, add this to your .cargo/config.toml for this project:

installing
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/yqznvrink36rzg4ry493lqqldr71jn82-orjson-3.8.4-vendor.tar.gz
strip is /nix/store/wn31i3dzwahz6ccws8bs1nwyqrpgsvj7-gcc-wrapper-11.3.0/bin/strip
patching script interpreter paths in /nix/store/yqznvrink36rzg4ry493lqqldr71jn82-orjson-3.8.4-vendor.tar.gz
checking for references to /build/ in /nix/store/yqznvrink36rzg4ry493lqqldr71jn82-orjson-3.8.4-vendor.tar.gz...
error: hash mismatch in fixed-output derivation '/nix/store/l2ivjcs7xmifvcimi7p06wymc8vf29kj-orjson-3.8.4-vendor.tar.gz.drv':
         specified: sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
            got:    sha256-O2W9zO7qHWG+78T+uECICAmecaSIbTTJPktJIPZYElE=
error: 1 dependencies of derivation '/nix/store/pcv2ddfb3hk2h5d991vd4knwl45nbl01-python3.10-orjson-3.8.4.drv' failed to build
error: 1 dependencies of derivation '/nix/store/9ka7ch657qimjvg2npf5g5mr3br4vwyl-python3-3.10.9-env.drv' failed to build
```

</details>

### Additional context

`shell.nix`:
```nix 
{ pkgs ? import <nixpkgs> { }
, poetry2nix ? import ~/dev/poetry2nix { }
}:
let
  myAppEnv = poetry2nix.mkPoetryEnv {
    projectDir = ./.;
    editablePackageSources = {
      my-app = ./src;
    };
  };
in myAppEnv.env
```

`pyproject.toml`:
```toml[tool.poetry]
name = "foo"
version = "0.1.0"
description = ""
authors = ["Raphael Das Gupta <raphael.dasgupta@ost.ch>"]
readme = "README.md"

[tool.poetry.dependencies]
python = "^3.10"

[tool.poetry.group.dev.dependencies]
orjson = "^3.8.4"

[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"
```

`poetry.lock`:
```toml
# This file is automatically @generated by Poetry and should not be changed by hand.

[[package]]
name = "orjson"
version = "3.8.4"
description = "Fast, correct Python JSON library supporting dataclasses, datetimes, and numpy"
category = "dev"
optional = false
python-versions = ">=3.7"
files = [
    {file = "orjson-3.8.4-cp310-cp310-macosx_10_7_x86_64.whl", hash = "sha256:94841c3418b818c8cda01ad632f0e740dad3a5bcbc38609f1fd163f4bd202ef9"},
    {file = "orjson-3.8.4-cp310-cp310-macosx_10_9_x86_64.macosx_11_0_arm64.macosx_10_9_universal2.whl", hash = "sha256:ffb9367d63a4c71cd0432fc5a4e295b0e89b061e90be9f5ff3ffca301ba2dd84"},
    {file = "orjson-3.8.4-cp310-cp310-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:74c25f71a41a3a213bee02f496abbc9ebc7cb0c8d1e5c1a5ab77c87b427b3f07"},
    {file = "orjson-3.8.4-cp310-cp310-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:30294eeb7417eca85572b64278f5c4fdd72049725d7d23da5042a61c1a2298c9"},
    {file = "orjson-3.8.4-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:36257f254635da390a9d689214c4cec9ec4ea224c5e2bcdfe97d15a384695be5"},
    {file = "orjson-3.8.4-cp310-cp310-manylinux_2_28_x86_64.whl", hash = "sha256:170a041d0d967e348ab086f1c9765846658cb0e807abe98e8320f665b1c67e9b"},
    {file = "orjson-3.8.4-cp310-cp310-musllinux_1_1_aarch64.whl", hash = "sha256:38df475523ca0f3f88b871821a9fb1a1437dad7c637847e0e7378024c7654473"},
    {file = "orjson-3.8.4-cp310-cp310-musllinux_1_1_x86_64.whl", hash = "sha256:54ade013afcda2f32f98227f0eb3dba4267c6efe7a8abd93bfec6d7af2546370"},
    {file = "orjson-3.8.4-cp310-none-win_amd64.whl", hash = "sha256:b0a8273ebcf363042e7847faaf67f92178aceb3476e374da5432f9e6772c7a80"},
    {file = "orjson-3.8.4-cp311-cp311-macosx_10_7_x86_64.whl", hash = "sha256:4c3c3571118371a5b88a89a0a157b4e98827d3a540c3f683c59d80e71f66195d"},
    {file = "orjson-3.8.4-cp311-cp311-macosx_10_9_x86_64.macosx_11_0_arm64.macosx_10_9_universal2.whl", hash = "sha256:86f71f2abc24b79e74a13f8d4eebb7d03cad43d6a46e2cad6fd5891423f84773"},
    {file = "orjson-3.8.4-cp311-cp311-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:1bff66b99ce8ba3d3b39ce10f27a848b3b68d80f64d77b8118d4648fafd2f964"},
    {file = "orjson-3.8.4-cp311-cp311-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:49c59a728510a6ef20de59f6262d1415108b323996634d263e4d0d613994fc13"},
    {file = "orjson-3.8.4-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:27225338f24a6906d254c4b8b3849135af758af55ae1f43e5ccbebcf2097628a"},
    {file = "orjson-3.8.4-cp311-cp311-manylinux_2_28_x86_64.whl", hash = "sha256:0c42fc0ed3f12bc7d0fd61a26899a8ac32deb6622289110b6d7f4d60d47dcff3"},
    {file = "orjson-3.8.4-cp311-none-win_amd64.whl", hash = "sha256:ad6d533ab9ef8b141d6749e7591ee3a3195037c217818e3b3351df1468efc012"},
    {file = "orjson-3.8.4-cp37-cp37m-macosx_10_7_x86_64.whl", hash = "sha256:654e219ace2c942f9d8f3a69935991f8c88efb4a85b074ba917960c800a1e5c8"},
    {file = "orjson-3.8.4-cp37-cp37m-macosx_10_9_x86_64.macosx_11_0_arm64.macosx_10_9_universal2.whl", hash = "sha256:0b57cc7a94b133140c85c3873d22e7a1f8e0d2489619a7e2e8640f9a9edfbc29"},
    {file = "orjson-3.8.4-cp37-cp37m-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:447a4c95f849872ac9af137153e8748b3b35d199c03a3bc618da6a3a23681f96"},
    {file = "orjson-3.8.4-cp37-cp37m-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:5d47002a88f60a35901925d4085a313647229950d2818044520eb0e30fc9447b"},
    {file = "orjson-3.8.4-cp37-cp37m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:e26422430227e2a8a6d07daee5390fb6510ce67f2b45a1016595e0ca8fb9515b"},
    {file = "orjson-3.8.4-cp37-cp37m-manylinux_2_28_x86_64.whl", hash = "sha256:77034415ceb563d38b5aa9713b8945faed7a5ddf5553a5e0831cee362b81e0b2"},
    {file = "orjson-3.8.4-cp37-cp37m-musllinux_1_1_aarch64.whl", hash = "sha256:486ecbd6548232ee4d533effcd4dbf5cfc547738ce1791f6204e949f1214c0a0"},
    {file = "orjson-3.8.4-cp37-cp37m-musllinux_1_1_x86_64.whl", hash = "sha256:b68f2f7f61b2f85623e0cd02eea04ab358cba9a01370fa8f2e96f1b2b83705b7"},
    {file = "orjson-3.8.4-cp37-none-win_amd64.whl", hash = "sha256:8b18925638936104e1c2bdc555b508733ef83c4c9ac96c5ec6666af6df12c22d"},
    {file = "orjson-3.8.4-cp38-cp38-macosx_10_7_x86_64.whl", hash = "sha256:c834eed62fd184d8c0a707f7d1dbaf57815c4865091bfc220182c83321c773f5"},
    {file = "orjson-3.8.4-cp38-cp38-macosx_10_9_x86_64.macosx_11_0_arm64.macosx_10_9_universal2.whl", hash = "sha256:6b07c1d76db95706e78e904f2329796d0c26ffed5eaea7a1be753ef7032c30dc"},
    {file = "orjson-3.8.4-cp38-cp38-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:3cb431a8168510458fa9d3a1a3ed8f56909187db953138080e8c8ef44484adde"},
    {file = "orjson-3.8.4-cp38-cp38-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:64d60e21096de30101348de25f20c516289365c056d0987ec76c74f5073af122"},
    {file = "orjson-3.8.4-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:ceabb527e0641ff23d5ed89f3d9c22f3df2cbe967547d28ea7fdc09c3de1168d"},
    {file = "orjson-3.8.4-cp38-cp38-manylinux_2_28_x86_64.whl", hash = "sha256:16a00f10321b4afacd8ecbe239e4ac8c6a77eb442caeffb638c9a49a653dfa2c"},
    {file = "orjson-3.8.4-cp38-cp38-musllinux_1_1_aarch64.whl", hash = "sha256:e9c0fb49362e79e3db158b8d307ff53adb4a5bd2706e1a4372393a4d661000fb"},
    {file = "orjson-3.8.4-cp38-cp38-musllinux_1_1_x86_64.whl", hash = "sha256:e7f72c7a810977b94d0411a5edf3b0eab7bc5763089ad0a0d639e49f651a0d91"},
    {file = "orjson-3.8.4-cp38-none-win_amd64.whl", hash = "sha256:2e6b088d6a1700fad0ce18715ec309bb7efc8ec6275413e1495261a3bbfbc17a"},
    {file = "orjson-3.8.4-cp39-cp39-macosx_10_7_x86_64.whl", hash = "sha256:5d618f30784321361927a65a6da484ef6da218f62c594987bd4b79f97df55036"},
    {file = "orjson-3.8.4-cp39-cp39-macosx_10_9_x86_64.macosx_11_0_arm64.macosx_10_9_universal2.whl", hash = "sha256:42aee49489de6aa3e9b3280c9a124d2f4a984be7a714cd8b13b27634af817521"},
    {file = "orjson-3.8.4-cp39-cp39-manylinux_2_17_aarch64.manylinux2014_aarch64.whl", hash = "sha256:31f919b370159b3d0c3f38844d66545a4520bc2652c32b9127d02e94f107247e"},
    {file = "orjson-3.8.4-cp39-cp39-manylinux_2_17_armv7l.manylinux2014_armv7l.whl", hash = "sha256:5fd174091a24a5ded94eaa141b1c989ce9dd6a7af8eadd525ef93dc5849cb48e"},
    {file = "orjson-3.8.4-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl", hash = "sha256:e030e8e50e908a7f8ba02b69c5692e30d9687ad184f0b75bfd8cd57df695093a"},
    {file = "orjson-3.8.4-cp39-cp39-manylinux_2_28_x86_64.whl", hash = "sha256:22b4c6066c1906b8d2a88cec8e8dc138ba2e08b17971419672734ca3ddab1995"},
    {file = "orjson-3.8.4-cp39-cp39-musllinux_1_1_aarch64.whl", hash = "sha256:90ea42189d81dea219dfda58374e2d9f07e1d17052fa33ade07441e5e50d62d2"},
    {file = "orjson-3.8.4-cp39-cp39-musllinux_1_1_x86_64.whl", hash = "sha256:53cc5be7c5c60d7583d71144bdb77171183f015b755ad5ab4a248fc03e61dad7"},
    {file = "orjson-3.8.4-cp39-none-win_amd64.whl", hash = "sha256:c2664d39fa599beb70afe8aa92c7ca905207a0b0aeb7597c038cb42394bd1a0e"},
    {file = "orjson-3.8.4.tar.gz", hash = "sha256:9dcb8d788254936de2791d6d2c29bc41a3dc2c9d9cb4f01c06c0e6d424844593"},
]

[metadata]
lock-version = "2.0"
python-versions = "^3.10"
content-hash = "cca3155746567834d70be28a31d46363ae8246badf06441d37b46130bfd81f9f"
```

**Comment by takeda** [comment]
I think you issue might have been fixed by https://github.com/nix-community/poetry2nix/pull/929 you might need to pull poetry directly instead of using the one packaged with nixpkgs (as there's slight delay).

**Comment by das-g** [comment]
This is indeed fixed on current `master`(449e238044b9eaa9e96695a364ac3a7a37ff6ffd). Closing.
